CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # This file is part of covfie, a part of the ACTS project
 #
-# Copyright (c) 2022-2023 CERN
+# Copyright (c) 2022-2024 CERN
```

Comment:
Note that I'll be removing all of these year changes very soon; please undo the changes in these copyright statements.

Reply:
Sure, I can revert the date changes... 🤷